plugins/git/commands

```diff
@@ -14,6 +14,7 @@ git_build_app_repo() {
   chmod 755 $TMP_WORK_DIR
   unset GIT_DIR GIT_WORK_TREE
   pushd $TMP_WORK_DIR > /dev/null
+  [[ ! -d "$DOKKU_ROOT/$APP" ]] && dokku apps:create $APP
```

Comment:
Otherwise we won't actually call the hook on every app creation. If I push a repo up without first creating the app, it skips the above...